properties/color/base.json

```
@@ -22,19 +22,28 @@
       "value": "#ffffff"
     },
     "orange": {
-      "value": "#ff7f3f"
+      "value": "#ff8912"
     },
     "coral": {
       "value": "#ff5d5f"
     },
-    "strawberry": {
-      "value": "#ec0b3c"
+    "tile": {
+      "value": "#ca3d3d"
     },
     "error": {
-      "value": "#cd0000"
+      "value": "#dc0000"
     },
     "melon": {
       "value": "#7bcc1f"
```

He hablado con Miriam, parece que el Zeplin se había actualizado mal y había algún color que no se veía cambiado, es el caso de el melon:
```suggestion
      "value": "#60a90e"
```